### ISSUE TITLE
❌ Software Removal | Pydio

### ISSUE THREAD

**Issue by dawidpotocki** [privacytoolsIO/privacytools.io#1331]
TL;DR Pydio is a mess.

### Desktop/Server

The application that they link from their site, `pydio-sync`
(https://github.com/pydio/pydio-sync) requires Python 2.7, which will no
longer be supported at the start of 2020. Last commit was on Sep 10, 2018
and `README.md` says that it is "Work in progress". The application is not
in the PyPI repository, so they recommend you to install the latest git
commmit by `pip install git+https://github.com/pydio/pydio-sync.git`. This
application isn't compatible with Pydio Cells, only Pydio 8.

There is also `cells` application, which is written in Go, but there are no
binary builds for any platform and the Windows version "might still have
unknown glitches and is not yet supported". So you have to install by
building it from the source. It is not compatible with Pydio 8, only Pydio
Cells.

None of these are really packaged in any */Linux or *BSD system, just look
at https://repology.org/projects/?search=pydio, it is only in AUR, which is
a community repository for Arch Linux. Also in AUR that is the old Python
2.7 version, not the `cells`. Repositories for Debian/Ubuntu systems are
only for old, mostly not suppored anymore versions like Ubuntu 17.10, 17.04
and sure, there is repo for supported Debian 9, but current release is
Debian 10. For CentOS/RHEL systems, they also only give repositories for
CentOS/RHEL 7, but there is already RHEL 8 and CentOS will also be updated
soon.

### Android

The application they link from their site, `Pydio Cells` is not compatible
with the Pydio 8 server, so if you have to, they recommend you to use Pydio
or Pydio Pro applications, which are were not updated since 2015.
Application has really bad reviews on Google Play, Cells has 3.1 stars and
only 500-999 installs and old Pydio has 3.5 stars and 50,000-99,999
installs. It is not available on F-Droid.

### iOS

The application they link from their site, `Pydio Lite`
(https://apps.apple.com/app/pydio/id709275884) is "deprecated in favor of
Pydio Pro", which I guess is https://apps.apple.com/app/pydio/id1109419882.
I assume that Pydio Lite is compatible with Pydio 8, but Pydio is just
compatible with Cells. Both have shitty reviews, 2.1 stars for Pydio Lite
and 2.7 stars for Pydio.
